constraint_utils/constraint_basics.py

```diff
@@ -191,12 +191,42 @@ def split_context_mt(raw_context_ht_path: str, coverage_ht_paths: Dict[str, str]
 
 def pre_process_data(ht: hl.Table, split_context_ht_path: str,
                      output_ht_path: str, overwrite: bool = False) -> None:
+    """ 
+    Join gnomAD and VEP context Tables.
```

Comment:
```suggestion
    Add annotations from VEP context Table onto gnomAD data.
```
sorry, just realized this was clearer. it might also be helpful to say exactly which annotations from the context table get added here also